From b4ac4170f4e0ea0daa9a9c0abe34cbe83445a4ac Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Tue, 14 Feb 2017 18:23:33 +0200
Subject: [PATCH 0819/1345] Documentation: bindings: net: mvpp2x: Add 5G port
 speed description

commit  e5e1695089c35d4648fb6bd0ce81e898851e07d4 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

This patch add 5Gbits port speed description to marvell-pp2x.txt
device tree bindings.

Change-Id: Iecdc567ff0fcf26cbc8096d5fe9d2a5578786fbe
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/37129
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/net/marvell-pp2x.txt       |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/net/marvell-pp2x.txt b/Documentation/devicetree/bindings/net/marvell-pp2x.txt
index 5c982ab..7b163dd 100644
--- a/Documentation/devicetree/bindings/net/marvell-pp2x.txt
+++ b/Documentation/devicetree/bindings/net/marvell-pp2x.txt
@@ -38,7 +38,8 @@ emac required properties:
 - interrupts: Link change interrupt for the mac.
 - port-id: Should be '0', '2', or '3' for ethernet macs, references the mac-id (GOP port-id).
 - phy-mode: See ethernet.txt file in the same directory
-- phy-speed: For 2.5Gbits SGMII port 2500 phy-speed should be set(used only for 2.5Gbits SGMII port)
+- phy-speed: For 2.5Gbits SGMII port 2500 phy-speed should be set.
+	     For 5Gbits XSI/SFI port 5000 phy-speed should be set(used only for 2.5Gbits and 5Gbits)
 - phy-names: Used in MAC's connected to comphy. See phy-mvebu-comphy.txt in the phy directory
 - phys: Used in MAC's connected to comphy. Comphy index, see  phy-mvebu-comphy.txt in the phy directory
 
-- 
1.7.9.5

